TaylorFramework/Modules/caprices/ProccessingMessage/PathFormatter.swift

```diff
@@ -18,12 +18,11 @@ private let SearchIndicator = ".*"
 private let CurrentDirectorySymbol = "."
 private let ParentDirectorySymbol = ".."
 private let RecursiveSymbol = "*"
-private let IgnoredPathPrefix = "**"
 
 extension Path {
     
     func lastComponentFromPath() -> String {
-        if self.isEmpty { return "" }
+        if self.isEmpty { return EmptyString }
```

Comment:
Can you return `self` if it is empty and get rid of `EmptyString` declaration. You have it declared as `String.Empty` anyway.